src/Microsoft.Azure.SignalR/HubHost/INegotiateHandler.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Connections;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal interface INegotiateHandler
```

Comment:
no need if not used in DI?

Reply:
relate to https://github.com/Azure/azure-signalr/pull/1103/files#r522594057